tasks/src/tabular-regression/about.md

```diff
@@ -12,6 +26,13 @@ Tabular regression is the task of predicting a numerical value given a set of at
 
 ## Useful Resources
 
+- [Kaggle Tabular Playground Series](https://www.kaggle.com/search?q=tabular+playground+in%3Acompetitions+sortBy%3Adate+competitionEvaluationAlgorithm%3ARMSE+competitionEvaluationAlgorithm%3ASMAPE+competitionEvaluationAlgorithm%3AMAE+competitionEvaluationAlgorithm%3AMCRMSLE) comes up with a new tabular challenge every month.
+- Tabular Regression using [FastAI](https://walkwithfastai.com/Regression_and_Permutation_Importance).
+
+- [Data Science for tabular data: Advanced Techniques](https://www.kaggle.com/code/vbmokin/data-science-for-tabular-data-advanced-techniques/notebook)
+
+- Tutorials on popular tree based methods like [XGBoost](https://machinelearningmastery.com/xgboost-for-regression/), [LightGBM](https://machinelearningmastery.com/light-gradient-boosted-machine-lightgbm-ensemble/) and [CatBoost](https://www.kaggle.com/code/allunia/house-prices-tutorial-with-catboost/notebook) to name a few.
```

Comment:
We try to avoid using terms, and we explain them if we have to to stay beginner friendly. 🙂

Reply:
I have updated the changes [here](https://github.com/huggingface/hub-docs/pull/434/commits/b45cc5850619518d70f17653abb205996ae5a907).
Removed this and simplified the list of resources keeping in mind the beginner to intermediate readers.